src/providers/assets/helpers.ts

```diff
@@ -4,11 +4,11 @@ import {
   BasicChannelFieldsFragment,
   VideoFieldsFragment,
 } from '@/api/queries'
-import { AssetType } from '@/hooks'
 import { createStorageNodeUrl } from '@/utils/asset'
 import { Logger } from '@/utils/logger'
 
-import { AssetData } from './types'
+import { AssetType } from './constants'
```

Comment:
Can we move `AssetType` to `types` after all? I know that we can use enums as a value, but I'm getting the impression that we have already too many files in this folder.

Reply:
Yes, good idea